### PR TITLE
Replace host composer installation with one from docker image

### PR DESCRIPTION
**Pull request by mateuszsip**
Now composer is run from host, so everybody needs local php installation with all required extensions.

The same thing should be done with tests.

### CODE REVIEW

**Comment by mateuszsip** [comment]
Looks like travis configuration has to be changed

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ help:
 	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST) | sort
 
 build: ## Build project
-	composer install
+	docker-compose exec -T php-fpm composer install
 
 test: ## Run tests
 	bin/phpspec run
```
Comment (matigda) [inline Makefile:11]: I guess then this also goes through docker.

**Comment by matigda** [comment]
@kejwmen yep definitely travis configuration needs to be changed :)

**Comment by mateuszsip** [comment]
#18 changed travis config, config for running tests will go in a separate PR.

**Review by matigda — APPROVED** [review]

**Pull request merged**